src/Tribe/iCal.php

```diff
@@ -275,8 +275,8 @@ public function generate_ical_feed( $post = null, $echo = true ) {
 			$item[] = 'CREATED:' . $tzoned->created;
 			$item[] = 'LAST-MODIFIED:' . $tzoned->modified;
 			$item[] = 'UID:' . $event_post->ID . '-' . $time->start . '-' . $time->end . '@' . parse_url( home_url( '/' ), PHP_URL_HOST );
-			$item[] = 'SUMMARY:' . str_replace( array( ',', "\n", "\r", "\t" ), array( '\,', '\n', '', '\t' ), html_entity_decode( strip_tags( $event_post->post_title ), ENT_QUOTES ) );
-			$item[] = 'DESCRIPTION:' . str_replace( array( ',', "\n", "\r", "\t" ), array( '\,', '\n', '', '\t' ), html_entity_decode( strip_tags( $event_post->post_content ), ENT_QUOTES ) );
+			$item[] = 'SUMMARY:' . str_replace( array( ',', "\n", "\r", "\t" ), array( '\,', '\n', '', '	' ), html_entity_decode( strip_tags( $event_post->post_title ), ENT_QUOTES ) );
```

Comment:
Hi @KZeni!

I understand why you are making this change, but this effectively has us searching for tab characters and replacing them with tab characters. I feel like there's a simpler solution:

```php
// This...
str_replace( array( ',', "\n", "\t" ), array( '\,', '\n', '	' ), $text );

// Is no different to this...
str_replace( array( ',', "\n", "\t" ), array( '\,', '\n', "\t" ), $text );

// And can be simplified to this...
str_replace( array( ',', "\n" ), array( '\,', '\n' ), $text );
```

I simplified the code a little, but in essence we could remove the tabs from both arrays. Do you agree, would you be down to make this change?